sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -24,6 +24,64 @@ import Amount from '~/react/components/helpers/Amount';
 import plansStyles from './plans.module.css';
 import Skeleton from 'react-loading-skeleton';
 
+interface PricingColumnHeaderProps {
+  plan: PlanIntervalPricing;
+  selectedInterval: IntervalKeys;
+}
+
+const PricingColumnHeader = ({
+  plan,
+  selectedInterval
+}: PricingColumnHeaderProps) => {
+  const { config } = useFrontier();
+  const selectedIntervalPricing = plan.intervals[selectedInterval];
+  const showPerMonthPrice =
+    config?.billing?.showPerMonthPrice === true && selectedInterval === 'year';
+
+  const perIntervalLabel = showPerMonthPrice
+    ? 'per seat/month'
+    : `per seat/${selectedInterval}`;
+
+  const amount = showPerMonthPrice
+    ? selectedIntervalPricing.amount / 12
+    : selectedIntervalPricing.amount;
+
+  const actualPerMonthAmount = plan.intervals['month']?.amount || 0;
+  const showDiscount = showPerMonthPrice && actualPerMonthAmount > 0;
+  const discount = showDiscount
+    ? ((actualPerMonthAmount - amount) * 100) / actualPerMonthAmount
+    : 0;
+
+  return (
+    <Flex gap="small" direction="column">
+      <Flex align={'center'} gap={'small'}>
+        <Text size={4} className={plansStyles.planTitle}>
+          {plan.title}
+        </Text>
+        {showDiscount && discount > 0 ? (
+          <Flex className={plansStyles.discountText}>
+            <Text weight={500}>{discount * -1}%</Text>
```

Comment:
Should we do an integer division to come up with discount percentage? We don't usually have decimal discounts like 10.12% off.

Reply:
removed decimal from discount percentage